app/src/main/java/org/gdg/frisbee/android/activity/AppInviteDeepLinkActivity.java

```diff
@@ -134,7 +133,7 @@ public void onConnectionSuspended(int i) {
     }
 
     @Override
-    public void onConnectionFailed(ConnectionResult connectionResult) {
+    public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
```

Comment:
Is `@NonNull` a project convention? I usually would advise against it because this should be the default for defensive programming.